src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,150 @@
+package duke.main;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.ToDo;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
+public class Duke {
+    static ArrayList<Task> taskList = new ArrayList<Task>();
+
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+
+        String userInput;
+        String logo = " ____        _        \n"
```

Comment:
Although not necessary, the `logo` here can be considered a `constant` so you can further extract and refactor out.